linode/manager
# M3-5182: RQ-ify Images Pt. 3

## Description 📝
Complete the trilogy of PRs for the RQ-ify Images effort. 🎉  🚀 

High-level overview of this PR:

- RQ-ify the Upload Image flow
  - The only thing remaining in Redux is the `set-pending-upload` logic (not a good candidate for moving to RQ)
- RQ-ify Images in search
  -  the `getSearchEntities` piece was a bit challenging, but the best course of action that didn't feel like a messy anti-pattern was to just decouple Images and Linodes from being returned in there, and to instead pass them in where needed in `withStoreSearch`
- `images` removed from Redux store & Image-related files in `src/store` removed

## How to test 🧪
- Ensure the Upload Image, Capture Image, StackScript, and Linode Create flows all work as expected.
- Ensure that the Search functionality works as expected, particularly as it relates to Images and Linodes.
